public/stylesheets/help.css

```diff
@@ -0,0 +1,45 @@
+.topic-list {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.topic-list li {
+  list-style-type: none;
+}
+
+.topic-list .btn-info {
+  margin: 4px 2px 0 0;
+}
+
+.viewed,
+.singleTopic > .topic-true {
+  color: #fff;
+  background: #1D9E9E;
+  border: 1px #026917 solid;
+}
+
+
```

Comment:
Remove extra white space, and place the modal specific styles in its own file please.